nil/tests/regression/regression_test.go

```diff
@@ -243,7 +327,7 @@ func (s *SuiteRegression) TestNonRevertedErrDecoding() {
 	code, err := contracts.GetCode(contracts.NameTest)
 	s.Require().NoError(err)
 
-	payload := types.BuildDeployPayload(code, common.EmptyHash)
+	payload := types.BuildDeployPayload(code, common.Hash{0x03})
```

Comment:
Why is something different here?

Reply:
Just to reduce chance of collision between tests.